tests/legacy-cli/e2e/tests/misc/typescript-warning.ts

```diff
@@ -3,9 +3,13 @@ import { updateJsonFile } from '../../utils/project';
 
 export default async function () {
   // typescript@2.8.0-dev.20180320 is not part of the officially supported range in latest stable.
-  let unsupportedTsVersion = '2.8.0-dev.20180320';
+  const unsupportedTsVersion = '2.8.0-dev.20180320';
 
   await updateJsonFile('src/tsconfig.app.json', configJson => {
+    configJson.compilerOptions = {
+      ...configJson.compilerOptions,
+      skipLibCheck: true,
```

Comment:
does this have any bearing on whether users would also need to skip type-checks of `.d.ts` files?

Reply:
They will still get the `ERROR in The Angular Compiler requires TypeScript >=3.1.1 and <3.2.0 but 3.0.3 was found instead.` but also a error from `tsc` because of some typings in Angular Core are not compatible with this version of TypeScript.

It shouldn't be turned on for users, as in reality they are being informed that they are using an unsupported version.